### PR TITLE
REL-4821: Add an Igrins2 tab to the OT Browser

### PR DESCRIPTION
**Pull request by cquiroz**

### CODE REVIEW

File: bundle/edu.gemini.pot/src/main/scala/edu/gemini/spModel/gemini/igrins2/Igrins2.scala [hunk 1]
```diff
@@ -242,6 +243,9 @@ object Igrins2 {
     SLIT_VIEWING_PROP
   ).map(p => (p.getName, p))
 
+  val getInstConfigInfo: JList[InstConfigInfo] =
+    List[InstConfigInfo]().asJava
```
Comment (swalker2m) [inline bundle/edu.gemini.pot/src/main/scala/edu/gemini/spModel/gemini/igrins2/Igrins2.scala:247]: A placeholder for now?

File: bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/inst/SciAreaFeature.java [hunk 2]
```diff
@@ -248,6 +250,11 @@ public void reinit(TpeImageWidget iw, TpeImageInfo tii) {
                 _ghost2Feat = new SciAreaPlotFeature(GhostScienceAreaGeometry$.MODULE$);
             }
             _feat = _ghost2Feat;
+        } else if (inst instanceof Igrins2) {
+            if (_igrins2Feat== null) {
+                _igrins2Feat = new SciAreaPlotFeature(GhostScienceAreaGeometry$.MODULE$);
```
Comment (swalker2m) [inline bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/inst/SciAreaFeature.java:255]: Is using GHOST intentional?  Is the science area feature required for the OT Browser?

Comment (cquiroz) [reply bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/inst/SciAreaFeature.java:255]: Ups this belongs to another PR 😄

File: bundle/edu.gemini.pot/src/main/scala/edu/gemini/spModel/gemini/igrins2/Igrins2.scala (hunk 1 above)
Comment (cquiroz) [reply bundle/edu.gemini.pot/src/main/scala/edu/gemini/spModel/gemini/igrins2/Igrins2.scala:247]: The requirement is to not have any controls on the browser yet

**Review by swalker2m — APPROVED** [review]
Looks great!  :+1:

**Pull request merged**